javalib/src/main/scala/java/util/concurrent/ThreadLocalRandom.scala

```diff
@@ -28,7 +28,7 @@ class ThreadLocalRandom extends Random {
       throw new IllegalArgumentException()
 
     val difference = bound - least
-    if (difference > 0) {
+    if (difference >= 0) {
```

Comment:
The equals condition can never occur, since it would require `least == bound`, which is illegal.